akka-docs/rst/scala/stream/stages-overview.rst

```diff
@@ -587,6 +587,17 @@ when the predicate first return false and then complete.
 
 **completes** when predicate returned false or upstream completes
 
+takeUntil
+^^^^^^^^^
+Pass elements downstream until a predicate function returns false, and then emit that element and complete. This
+differs from takeWhile in that it's inclusive of the final element that causes the predicate to reutrn false
+
+**emits** while the predicate is true and until the first false result
+
+**backpressures** when downstream backpressures
+
+**completes** when predicate returned false, after emitting that final element, or upstream completes
+
```

Comment:
Great but there is another one where you need to add it: `akka-docs/rst/java/stream/stages-overview.rst`